tests/percy/photographer.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Responsible for navigating pages and capturing snapshots
+ */
+class Photographer {
+  /**
+   * Navigate to a universal page
+   * 
+   * @param {Object} obj
+   * @param {string} obj.queryParams params to be put on the page. A leading '?' or '&' should
```

Comment:
We could potentially make the interface of this and the other methods friendlier by accepting a `queryParams` object and doing the serialization internally. What do you think?

Reply:
sure, I like that